freerdp-3.yaml

```
@@ -0,0 +1,121 @@
+package:
+  name: freerdp-3
+  version: 3.0.0
```

```suggestion
  version: 3.1.0
```
There is a new version available.